uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/contract-entitlements.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-env amd */
+
+define([
```

Comment:
@deb1990 please confirm that we discussed what I tried to explain in the comment above and everything is clear now